frontend/src/components/IstioWizards/ServiceWizard.tsx

```diff
@@ -806,15 +807,15 @@ export class ServiceWizard extends React.Component<ServiceWizardProps, ServiceWi
     const titleAction =
       this.props.type.length > 0
         ? this.props.update
-          ? `Update ${WIZARD_TITLES[this.props.type]}`
-          : `Create ${WIZARD_TITLES[this.props.type]}`
+          ? `${t('Update')} ${WIZARD_TITLES[this.props.type]}`
+          : `${t('Create')} ${WIZARD_TITLES[this.props.type]}`
```

Comment:
Add the `t` function to the `WIZARD_TITLES` variable to be translated (explained in a comment below):
```suggestion
          ? `${t('Update')} ${t(WIZARD_TITLES[this.props.type])}`
          : `${t('Create')} ${t(WIZARD_TITLES[this.props.type])}`
```